pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
+
+const (
+	Nano  Scale = -9
+	Micro Scale = -6
+	Milli Scale = -3
```

Comment:
Correct about base-10 (will update Scale comment). As for larger values, it's an arbitrary decision but asking for smaller values seemed like the common case. I'll add up to `Exa` since thats what we have in suffixes.